src/test/java/reactor/pool/SimpleFifoPoolTest.java

```diff
@@ -466,6 +465,25 @@ void consistentThreadDeliveringWhenNoElementsAndFullAndRaceDrain(int i) throws I
             //we expect that only 1 element was created
             assertThat(newCount).as("elements created in round " + i).hasValue(1);
         }
+
+        @Test
+        @Tag("loops")
+        void acquireReleaseRaceWithMinSize_loop() {
+            AtomicInteger newCount = new AtomicInteger();
+            PoolConfig<PoolableTest> testConfig = from(Mono.fromCallable(() -> new PoolableTest(newCount.getAndIncrement())))
+                    .sizeBetween(4, 5)
+                    .buildConfig();
+            SimpleFifoPool<PoolableTest> pool = new SimpleFifoPool<>(testConfig);
+            final Scheduler racer = Schedulers.fromExecutorService(Executors.newFixedThreadPool(2));
```

Comment:
the thread pool is not closed after the test. Not critical, but a resource leak

Reply:
good point, let's avoid introducing more tests that leak resources ;) updated